app/main.py

```diff
@@ -62,6 +59,16 @@ def get_value_set_by_id(id: str, version: str = None, code: str = None):
     }
     return get(url, params)
 
+# Get https://phinvads.cdc.gov/baseStu3/CodeSystem/ and return a whole code system result
+@app.get("/phinvads/CodeSystem")
+@cache(namespace="phinvads", expire=3600)
+def get_code_systems(code: str = None):
```

Comment:
According to section 3.2.2 of the docs, this endpoint also accepts `name`, `title`, and `identifier` params, I think similarly to the ValueSet bundle endpoint. It can also take a `_getpages` param for pagination. Can you add these? Otherwise looks good!

Reply:
I had these originally and then removed prior to PR 🤣 I'll add em back.